.github/ISSUE_TEMPLATE/04_bug.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug
+about: Report a bug
+title: (Set a clear title describing the issue)
+labels: 'untriaged'
+assignees: ''
+---
+  
+(Describe your issue and goal here)
+  
+
+#### The Emoji Manager Version
+
+(Specify the tag, branch, or commit hash)
+  
+#### Java Runtime version
+  
+(Paste the output of `java -version`)
+  
+#### OS info
+  
+(Paste the output of `sw_vers && uname -v` on macOS/Linux or `ver` on Windows OS)
+
+#### Database info
+
+(Specify the version of Postgres)
+
+#### Steps to reproduce:
+  
+(Share the commands to run, source code, screenshots, etc.)
+  
+  1.
+  2.
+  3.
+  
+#### Expected result:
+  
+(Describe what you expected to happen)
+  
+#### Actual result:
+  
+(Describe what actually happened with logs, screenshots)
+
+### Requirements
+
+Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you agree to those rules.
```

Comment:
Are these supposed to be links to Slack's SDK contribute guideline and code of conduct?

Reply:
Doh! I borrowed from Slack and missed a few..  will update.